### ISSUE TITLE
Import new site and sampling data: University of Michigan Biological Station (umbs_dn.us) 2021-2023 (Yr 0-2)

### ISSUE THREAD

**Issue by nutnetadmin** [nutnetadmin/dragnet-data-tracker#32]

**Comment by nutnetadmin** [comment]
Started entry but seven species IDs don't have corresponding taxa names in the taxa table. Emailed Karin Rand for resolution.

GRASPE, 2022-07-17
CIRSVUL, 2023-07-10
ARAGLA, 2023-07-10
ARTCAMP, 2023-07-10
TRADUB, 2023-07-10
ARESER, 2023-07-10
CLIVUL, 2023-07-10

**Comment by nutnetadmin** [comment]
From Karin Rand:

I have updated the spreadsheet from 2023 accordingly.
For quicker reference:
> CIRSVUL- Cirsium vulgare
> ARAGLA- Arabi glabra
> ARTCAMP, Artemisia campestris
> TRADUB= Tragopogon_dubius was listed in taxa as TRASPE so I replaced it with TRADUB
> CLIVUL- Clinopodium_vulgare

There was one ARESER in the cover data as I hadn’t correctly fixed my raw data before copying it over. This is instead CENSTO, I changed this in the cover data sheet.


As for 2022 GRASPE this is a key for GRASS_SPECIES which I assume is an unknown grass. I did not take data that year but that is what I have from our notes. 


PW - I placed a new 'corrected' version of the UMBS data in the 2023 raw data folder
